platform/security/pdp/security-pdp-authzrealm/pom.xml

```diff
@@ -111,8 +111,8 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Embed-Dependency>
-                            org.wso2.balana,
-                            commons-io
+                            arbitro-core,
+                            commons-io,
```

Comment:
They are both definitely used. I'm not sure how it was working either, but it was working.